hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/info/NodeInfo.java

```diff
@@ -30,5 +30,5 @@ public interface NodeInfo {
      *
      * @return whether this node has zero stake.
      */
-    boolean isSelfZeroStake();
+    boolean isSelfZeroWeight();
```

Comment:
Was this intended to change to weight? Weight is a platform level concept and should only be seen in the application layer when calculating from stake to provide the platform.

Reply:
Fixed